src/DIRAC/TransformationSystem/scripts/dirac_production_runjoblocal.py

```diff
@@ -159,11 +159,9 @@ def main():
             _downloadinputdata = False
             _jobID = switch[1]
 
-    from DIRAC.ConfigurationSystem.Client.Helpers.CSGlobals import Extensions
+    from DIRAC.ConfigurationSystem.Client.Helpers.CSGlobals import getCSExtensions
 
-    ext = Extensions()
-    _vo = ext.getCSExtensions()[0]
-    _diracPath = Extensions().getExtensionPath("DIRAC")
+    _vo = getCSExtensions()[0]
```

Comment:
Niavely I would expect this to be `getVO()`?

In fact `__configurePilot` seems to be overriding it with `getVO()`.

Reply:
I tried to simplify it, but not sure I understand how to test it (3bfb6d50c400afd4f0f029bea822d47d3c19a11d)

Reply:
From command description:
>This method was created specifically for LHCb pilots, more info
    about othe VOs is needed to make it more general.

So this is a special command for the LHCb? Maybe to make it more universal or move it to the LHCb extension?

Reply:
@fstagni Could you look when have a time, please?

Reply:
@fstagni For example the following line seems not universal, or ``ce.debug.ch`` CE - OK for all cases?

>"dirac-pilot.py -S %s -l %s -C %s -N ce.debug.ch -Q default -n DIRAC.JobDebugger.ch -dd"

Reply:
These values can be universal